modules/gapi/include/opencv2/gapi/garray.hpp

```diff
@@ -369,6 +369,7 @@ template<typename T> class GArray
 };
 
 using GArrayP2f = GArray<cv::Point2f>;
+using GRects    = GArray<cv::Rect>;
```

Comment:
Does it make sense to implement `wrapper` for `GArray` & `GOpaque` to avoid making `using` for every `GArray` & `GOpaque` type ?

Reply:
Do you really need `using` everywhere? And why does it bother you? I don't think you need an extra wrapper here

Reply:
It is impossible to wrap template object into python, so I just make using and wrap it in `shadow_gapi.hpp`

Reply:
Fixed